### PR TITLE
OI clean up.

### PR DESCRIPTION
**Pull request by deslobodzian**

### CODE REVIEW

File: src/main/java/org/frc5687/rapidreact/commands/catapult/IdleCatapult.java [hunk 1]
```diff
@@ -23,14 +23,8 @@ public void initialize() {
     @Override
     public void execute() {
         super.execute();
-        double speed = _oi.getWinchMotorSpeed();
-        metric("winch speed", speed);
         _catapult.setWinchMotorSpeed(0);
-//        double speedSpring = _oi.getSpringMotorSpeed();
-//        metric("spring speed", speedSpring);
-//        _catapult.setSpringMotorSpeed(0.0);
         _catapult.setSpringMotorSpeed(_oi.getSpringMotorSpeed());
```
Comment (BenBernardCIS) [inline src/main/java/org/frc5687/rapidreact/commands/catapult/IdleCatapult.java:27]: Do you still need manual spring motor control?

Comment (deslobodzian) [reply src/main/java/org/frc5687/rapidreact/commands/catapult/IdleCatapult.java:27]: oh, shoot. I thought that was holding position. So no.

File: src/main/java/org/frc5687/rapidreact/commands/catapult/IdleCatapult.java [hunk 2]
```diff
@@ -24,7 +24,6 @@ public void initialize() {
     public void execute() {
         super.execute();
         _catapult.setWinchMotorSpeed(0);
-        _catapult.setSpringMotorSpeed(_oi.getSpringMotorSpeed());
     }
```
Comment (BenBernardCIS) [inline src/main/java/org/frc5687/rapidreact/commands/catapult/IdleCatapult.java:27]: Sorry to nit-pick, but shouldn't this also set speed to 0 like you did for setWinchMotorSpeed?

Comment (deslobodzian) [reply src/main/java/org/frc5687/rapidreact/commands/catapult/IdleCatapult.java:27]: It shouldn't matter too much as motion magic is treated differently, ALSO IdleCatapult is never called.

**Review by BenBernardCIS — APPROVED** [review]

**Pull request merged**